core-tests/shared/src/test/scala/zio/FunctionIOSpec.scala

```diff
@@ -1,202 +1,143 @@
 package zio
 
-import zio.FunctionIO._
+import zio.FunctionIO.{ fail => _, _ }
```

Comment:
Any reason you're hiding this? Doesn't look like there is any import conflict and you use it later.

Reply:
There was a conflict earlier. I Forgot to simplify this import.